autoload/vimteractive.vim

```diff
@@ -1,9 +1,57 @@
+let s:iconn_cw_winnr = -1
```

Comment:
Is this variable used anywhere else?

Reply:
No, it's a leftover to my first attempt to implement `^W`-navigation. I'll remove it.